tests/testcases/config-variables/src/lib.rs

```diff
@@ -12,15 +12,21 @@ fn handle_request(req: Request) -> Result<Response> {
         .query()
         .expect("Should have a password query string");
     let query: std::collections::HashMap<String, String> = serde_qs::from_str(query)?;
-    let provided_password = query
+    let expected_password = query
         .get("password")
         .expect("Should have a password query string");
-    let expected_password = config::get("password")?;
+    let actual_password = config::get("password")?;
 
-    ensure!(
-        provided_password == &expected_password,
-        "password must match expected"
-    );
+    if expected_password != &actual_password {
+        let error_msg = format!(
+            "expected password: '{}', actual password: '{}'",
```

Comment:
As this is a test case, I suggest phrasing this in terms of the "what went wrong" e.g. "value we expected to find in configuration / value actually found in configuration".  (This also precludes problems of people imitating out of context: "well Fermyon said I should return the correct password in auth failure messages, for debugging you know.")

Reply:
done